dbt/adapters/athena/impl.py

```diff
@@ -26,6 +27,7 @@
 class AthenaAdapter(SQLAdapter):
     ConnectionManager = AthenaConnectionManager
     Relation = AthenaRelation
+    Column = AthenaColumn
```

Comment:
Do we still need this or is it covered by the macros ?

Reply:
Should be possible to drop that.

Reply:
Oops, sorry.
Thanks 🙏

Reply:
removed by [ca0c45a](https://github.com/dbt-athena/dbt-athena/pull/30/commits/ca0c45abf05a84e9708edc24e232a79ab10c48bd)